provisioning/plugins/corim-tpm-enacttrust-decoder/extractor.go

```diff
@@ -91,7 +91,7 @@ func (o Extractor) TaExtractor(avk comid.AttestVerifKey) (*endorsement.TrustAnch
 
 	// extract AK pub
 	if len(avk.VerifKeys) != 1 {
-		return nil, errors.New("expecting exactly one AK public key")
+		return nil, errors.New("expecting exactly one IAK public key")
```

Comment:
in TPM this is AK (i.e., attestation key).   In PSA it is IAK (i.e., initial attestation key)

```suggestion
		return nil, errors.New("expecting exactly one AK public key")
```